packages/runtime/src/next/components/tests/controls/color-control/client.test.tsx

```diff
@@ -0,0 +1,55 @@
+/** @jest-environment jsdom */
+
+import { Color, ColorData } from '@makeswift/controls'
+import { testPageControlPropRendering } from '../page-control-prop-rendering'
+import { http, HttpResponse } from 'msw'
+import { server } from '../../../../../mocks/server'
+import { swatchId, swatch, value, cacheData } from './fixtures'
+
+const swatchesBaseUrl = `/api/makeswift/swatches`
+
+describe.each([
+  {
+    version: 0,
+    toData: (value: ColorData) => value,
+  },
+  {
+    version: 1,
+  },
+])('color data v$version', ({ toData }) => {
```

Comment:
Client tests haven't changed (other than titles), but are quite a bit shorter due to moving fixtures into a separate file.